### DISCUSSION TITLE
is there any intention to implement d3 flamegraph instead of regular flamegraph

### DISCUSSION THREAD

**Discussion by avtaritet** [KDAB/hotspot#585] [Ideas]
hi Milian do you have any intention to implement d3 flamegraph  instead of regular flamegraph? 
I mean this one  https://github.com/spiermar/d3-flame-graph
Thanks

**Comment by beedell-roke** [comment]
> https://github.com/KDAB/hotspot/discussions/585#discussion-6036145

@avtaritet, what's the advantage of that?

**Comment by milianw** [comment]
No intention from my side, no. Patches welcome

**Comment by GitMensch** [comment]
@avtaritet: I _think_ that's about a "static" and a clickable framegraph, no?

... but the Hotspot one can already be "zoomed in and out" exactly the way the d3 flamegraph demo is rendered. ...so: What is the question and @milianw what would you welcome patches for?

I guess that's about an Export option (so along "Image" and "SVG",  we would have "interactive HTML").
That would indeed be nice (and seemingly a similar implemented to the SVG one.
